examples/00-fluent/frozen_rotor_workflow.py

```diff
@@ -0,0 +1,424 @@
+impeller_hub.relative = True
+impeller_hub.velocity_spec = "Rotational"
+
+#  inblock-shroud
+
+inblock_shroud = solver.settings.setup.boundary_conditions.wall[
+    "inblock-shroud"
+].momentum
+inblock_shroud.wall_motion = "Moving Wall"
+inblock_shroud.relative = False
+inblock_shroud.velocity_spec = "Rotational"
+
+################################################################################################################
+# Define Boundary Conditions
+# ==============================================================================================================
+
+# Inlet Boundary Condition
+pressure_inlet = solver.settings.setup.boundary_conditions.pressure_inlet["inlet"]
+pressure_inlet.momentum.supersonic_or_initial_gauge_pressure.value = -100
+
+# It seems, need to change the boundary condition to mass flow outlet
+solver.settings.setup.boundary_conditions.set_zone_type(
+    zone_list=["mass-flow-inlet-11"], new_type="mass-flow-outlet"
+)
+# Outlet Boundary Condition
+
+mass_flow_outlet = solver.settings.setup.boundary_conditions.mass_flow_outlet[
+    "mass-flow-inlet-11"
+]
+mass_flow_outlet.momentum.mass_flow_rate.value = 90  # kg/s
+
+
+# Create a turbo interfaces
+# enable the turbo models
+solver.settings.setup.turbo_models.enabled = True
+
+solver.settings.setup.mesh_interfaces.turbo_interface.create(
+    adjacent_cell_zone_1="all",
+    adjacent_cell_zone_2="all",
+    mesh_interface_name="nps",
+    paired_zones=[],
+    turbo_choice="No-Pitch-Scale",
+    zone1="interface-impeller-outlet",
+    zone2="interface-volute-inlet",
+)
+
+# Perform the mesh check
+solver.settings.mesh.check()
+
+################################################################################################################
+# Define Solver Settings
+# ==============================================================================================================
+methods = solver.settings.solution.methods
+methods.spatial_discretization.gradient_scheme = "green-gauss-node-based"
+methods.high_order_term_relaxation.enable = True
+
+################################################################################################################
+# Define Named Expressions
+# ==============================================================================================================
+
+pump_head = solver.settings.setup.named_expressions.create("head")
+pump_head.definition = "(({p-out} - {p-in}) / (998.2 [kg/m^3] * 9.81[m/s^2]))"
+pump_head.output_parameter = True
+
+
+################################################################################################################
+# Define Report Definitions
+# ==============================================================================================================
+
+# Create a report definition
+# p-out
+solver.settings.solution.report_definitions.surface.create("p-out")
+
+outlet_pressure_report_def = solver.settings.solution.report_definitions.surface[
+    "p-out"
+]
```

Comment:
```suggestion
outlet_pressure_report_def = solver.settings.solution.report_definitions.surface.create("p-out")
```
@abhishekchitwar `create()` returns the new object directly.